web/crux-ui/src/components/main/footer.tsx

```diff
@@ -11,11 +12,14 @@ const Footer = (props: React.HTMLProps<HTMLDivElement>) => {
     <footer className={clsx('font-poppins items-center flex py-6 justify-between', className)} {...forwardProps}>
       <div className="flex items-stretch text-sm">
         <span className="text-light pr-2">Copyright © {new Date().getFullYear()}</span>
-        <span className="text-dyo-turquoise font-bold">dyrector.io</span>
       </div>
       <div className="flex items-stretch text-sm">
-        <span className="text-light pr-2">Hand-crafted &amp; Made with</span>
-        <Image src="/heart.svg" alt={t('heart')} width={20} height={20} />
+        <span className="text-light pr-2">
+          <Link href="https://github.com/dyrector-io/dyrectorio" passHref target={'_blank'}>
+            <a><span className="text-dyo-turquoise font-bold">dyrector.io</span></a>
+          </Link>{' '}
+          is an Open Source project
```

Comment:
This could be added to the common localization.

Reply:
Good point, I'll fix it - thanks!